bitnami/solr/templates/_helpers.tpl

```diff
@@ -0,0 +1,92 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "solr.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "solr.fullname" -}}
+{{- include "common.names.fullname" . -}}
+{{- end -}}
+
+{{/*
+Define the name of the client service for solr
+*/}}
+{{- define "solr.service-name" -}}
+{{- printf "%s-%s" (include "solr.fullname" .) "svc" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Define the name of the solr exporter
+*/}}
+{{- define "solr.exporter-name" -}}
+{{- printf "%s-%s" (include "solr.fullname" .) "exporter" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+The name of the zookeeper headless service
+*/}}
+{{- define "solr.zookeeper.service" -}}
+{{- printf "%s-%s" .Release.Name "zookeeper" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "solr.imagePullSecrets" -}}
+{{ include "common.images.pullSecrets" (dict "images" (list .Values.image ) "global" .Values.global) }}
+{{- end -}}
+
+
+{{/*
+Return the proper Apache Solr image name
+*/}}
+{{- define "solr.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Solr Exporter image name
+*/}}
+{{- define "exporter.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.exporter.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+ Create the name of the service account to use
+ */}}
+{{- define "solr.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{ default (include "solr.fullname" .) .Values.serviceAccount.name }}
```

Comment:
For consistency, I'd add `{{-` and `-}}` to all the helper lines. That's what we usually do in other charts